pages/add/add.scss

```diff
@@ -51,15 +59,47 @@
 
   fieldset.thumbnail {
     .help-text {
-      margin-left: 1rem;
-      color: $gray-light;
+      @include form-helper-text("");
+
+      @media (min-width: $bp-sm) {
+        margin-left: 1rem;
+      }
     }
 
     input[type="button"] {
       @include thumbnail-button-style();
     }
   }
 
+  fieldset.get_involved {
+    textarea {
+      height: 4.625rem;
```

Comment:
Why REMs here? Seems like EMs would be better, as it’s a direct mapping to “lines of text to fit”.

Reply:
`rem`s were originally used to establish `textarea` height in this file, so primarily for consistency. I can update them to `em`s though.